examples/example_MLP_model.cc

```diff
@@ -520,8 +520,8 @@ TEST_F(ProductionModel, 1LUT) {
   auto D = FLAGS_D;
   auto L1 = FLAGS_L1;
   auto E1 = FLAGS_E1;
-  auto options = tc::MappingOptions::makeNaiveMappingOptions()
-                     .tile({1, 32})
+  auto options = tc::CudaMappingOptions::makeNaiveCudaMappingOptions()
+                     .tile(1, 32)
```

Comment:
Just because I dropped initializer list for tile (coming from forwarding macros).
I could work harder to bring initializer list back but at this point with the variadic template it feels unnecessary.
We could also for consistency drop initializer list everywhere, they don't play as nicely with templates.